index.html

```diff
@@ -219,23 +219,22 @@ <h4 class="ibm-padding-bottom-0 ibm-padding-top-0">Application closing soon. Sub
 
                                         <div class="ibm-col-12-4 ibm-col-medium-12-6">
                                             <div class="ibm-card ibm-card--noborder">
-                                                <a href="https://ibm.biz/CIO-London-Full-Stack-Web-Developer" id="Full-Stack-Web-Developer" target="_blank" rel="noopener noreferrer" class="ibm-blocklink ibm-padding-content">
+                                                <a href="https://careers.ibm.com/ShowJob/Id/708719/Full-Stack-Web-Developer/?lang=en" id="Full-Stack-Web-Developer" target="_blank" rel="noopener noreferrer" class="ibm-blocklink ibm-padding-content">
```

Comment:
I've asked the rest of the team if we can get the short URLs updated so these could then be left as is - makes it easier for the recruitment team to keep them updated without requiring as many changes from us!

I'll get back to you on it :+1:

Reply:
ok thank you.  Will look forward to hearing back on that.